pkgs/by-name/ha/handheld-daemon/package.nix

```diff
@@ -5,6 +5,7 @@
 , lib
 , python3
 , toybox
+, coreutils
```

Comment:
[`chmod` is already provided by `toybox`](https://landley.net/toybox/help.html).